suricata.yaml.in

```diff
@@ -1281,6 +1287,9 @@ defrag:
 
 flow:
   memcap: 128mb
+# memcap-policy: ignore       # Can be "drop-flow", "pass-flow", "bypass",
```

Comment:
I think these comment blocks inside the actual settings are a bit of an anti-pattern. With stream we have a block above it.